### ISSUE TITLE
Tile: Text illegible on dark theme

### ISSUE THREAD

**Issue by phkuo** [OfficeDev/office-ui-fabric-react#11291]
Broken code:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/Tile/Tile.scss

The text on a tile in a dark theme is illegible:
![image](https://user-images.githubusercontent.com/20691641/69466193-32cf5680-0d38-11ea-9dcd-ef46f7d776d6.png)

The issue is that the text color uses `$ms-color-white`, while the background gradient uses a hardcoded black color (line 424). Themed colors and hardcoded colors should not be mixed. In this case, either we should find a way to make the gradient themable (which sounds hard), or also hardcode #fff as the font color (which sounds like the sane path forward).

**Comment by aneeshack4** [comment]
@phkuo Where is setting the text color? I see `ms-color-white` for `name` on line 476 - is that what you're referring to? Hardcoding the #fff as the font color sounds reasonable to me as well. Do you think some design input here could be useful?

**Comment by phkuo** [comment]
Not sure which line it is exactly. Design intent here is clear, i vote we just hardcode it.

Sent by phone

________________________________

**Comment by aneeshack4** [comment]
@phkuo I'm following up online.

**Comment by aneeshack4** [comment]
It seems either the `$ms-color-white` for [`name`](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/Tile/Tile.scss#L477) or inside [this `hasForegroundFrame`](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/Tile/Tile.scss#L148) are the most appropriate places for this change - I tried changing both of these separaetly,  Tile text looks just as visible in dark mode as in regular:
![image](https://user-images.githubusercontent.com/4161791/69768704-765e0200-1136-11ea-8675-a1d2f7f1ef36.png)
![image](https://user-images.githubusercontent.com/4161791/69768709-7958f280-1136-11ea-8244-1f9041b48836.png)
Tagging component owner @ThomasMichon - Thomas, do you have any additional insights into this?

**Comment by phkuo** [comment]
@aneeshack4 it's specifically for tiles on top of picture thumbnails, sorry if I didn't make that clear! In those cases, it adds a gradient at the bottom, and the text is on top of that gradient. The gradient is hardcoded to white, but the text is themed.

**Comment by aneeshack4** [comment]
will need some input from @ThomasMichon who is the component owner

**Comment by ThomasMichon** [comment]
The simple fix here is to switch to a direct color value for the text in the nameplate when in 'media' mode. The reason it seems to be okay in the Fabric demo app is that Tile *doesn't* actually pick up the dark theme set locally, so what you see doesn't represent what would happen if a global dark theme were to be applied.

**Comment by aneeshack4** [comment]
@ThomasMichon To clarify, do you mean we should add a media selector to Tile.scss and set a hex color value for text in that case? Which color value?
